theme/management/commands/email_users.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand
+from django.core.mail import send_mail
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from theme.models import UserQuota, QuotaMessage
+
+
+class Command(BaseCommand):

```

Comment:
Not sure how this command supposed to be used. It seems if this command is executed passing one more usernames, it will send email to those users irrespective of their current quota status. Should email be sent only to those who  are close to their allowed quota limit?